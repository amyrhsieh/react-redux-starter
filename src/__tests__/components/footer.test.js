import React from 'react';
import renderer from 'react-test-renderer';
import Footer from 'Components/Footer.js';

describe('Footer (Snapshot)', () => {
  it('Footer component renders footer', () => {
    const component = renderer.create(<Footer />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
