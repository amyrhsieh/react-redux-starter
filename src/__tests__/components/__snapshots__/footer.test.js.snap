// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer (Snapshot) Footer component renders footer 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <span
      className="text-muted"
    >
      Place sticky footer content here.
    </span>
  </div>
</footer>
`;
